scripts/sidecar-mysql/entry-point.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -xe
+trap '' SIGINT
+trap ''  SIGQUIT
+trap '' SIGTSTP
+
+cd /employees_db
+until mysql -h mysql -u root -p${MYSQL_ROOT_PASSWORD} -t < employees.sql
```

Comment:
MySQL server hostname should be configurable or passed as environment variable, otherwise one will have to create a service 'mysql' in the same namespace